src/sections/evidence/Chembl/Body.js

```diff
@@ -132,12 +133,29 @@ const columns = [
   {
     id: 'clinicalStatus',
     label: 'Status',
-    renderCell: ({ studyStopReason, clinicalStatus }) => {
+    renderCell: ({
+      studyStopReason,
+      clinicalStatus,
+      studyStopReasonCategories,
+    }) => {
       return clinicalStatus ? (
         studyStopReason ? (
           <Tooltip
             showHelpIcon
-            title={<>Study stop reason: {studyStopReason}</>}
+            title={
+              <>
+                Study stop reason: {studyStopReason}
+                <p>
```

Comment:
Is working perfectly. Can you just change this <p> tag for a container element <div> with a `display: inline-block` CSS prop